diff --git a/runcoms/zpreztorc b/runcoms/zpreztorc
index 4778944..8e8a063 100644
--- a/runcoms/zpreztorc
+++ b/runcoms/zpreztorc
@@ -32,6 +32,10 @@ zstyle ':prezto:load' pmodule \
   'spectrum' \
   'utility' \
   'completion' \
+  'git' \
+  'node' \
+  'syntax-highlighting' \
+  'history-substring-search' \
   'prompt'
 
 #
diff --git a/runcoms/zprofile b/runcoms/zprofile
index 08d39be..6cd89bc 100644
--- a/runcoms/zprofile
+++ b/runcoms/zprofile
@@ -17,8 +17,8 @@ fi
 # Editors
 #
 
-export EDITOR='nano'
-export VISUAL='nano'
+export EDITOR='vim'
+export VISUAL='vim'
 export PAGER='less'
 
 #
@@ -26,7 +26,7 @@ export PAGER='less'
 #
 
 if [[ -z "$LANG" ]]; then
-  export LANG='en_US.UTF-8'
+  export LANG='ja_JP.UTF-8'
 fi
 
 #
@@ -43,8 +43,14 @@ typeset -gU cdpath fpath mailpath path
 
 # Set the list of directories that Zsh searches for programs.
 path=(
-  /usr/local/{bin,sbin}
   $path
+  /usr/local/{bin,sbin}
+  /usr/local/pgsql/bin
+  /usr/local/subversion/bin
+  ~/bin
+  ~/.linuxbrew/bin
+  ~/.nodebrew/current/bin
+  ~/.config/composer/vendor/bin
 )
 
 #
diff --git a/runcoms/zshrc b/runcoms/zshrc
index 039b882..94feebd 100644
--- a/runcoms/zshrc
+++ b/runcoms/zshrc
@@ -11,3 +11,5 @@ if [[ -s "${ZDOTDIR:-$HOME}/.zprezto/init.zsh" ]]; then
 fi
 
 # Customize to your needs...
+umask 002
+eval $(/home/linuxbrew/.linuxbrew/bin/brew shellenv)
